x/datadeal/client/cli/txSellData.go

```diff
@@ -0,0 +1,49 @@
+package cli
+
+import (
+	"encoding/base64"
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/medibloc/panacea-core/v2/x/datadeal/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdSellData() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "sell-data [dealID] [verifiableCID]",
+		Short: "Sell data",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			sellerAddress := clientCtx.GetFromAddress().String()
+
+			dealID, err := strconv.ParseUint(args[0], 10, 64)
+			if err != nil {
+				return err
+			}
+
+			verifiableCID, err := base64.StdEncoding.DecodeString(args[1])
```

Comment:
This is same too,

Reply:
Fixed in [9ed50b9](https://github.com/medibloc/panacea-core/pull/422/commits/9ed50b907de38b143625cb4846520db63da6d1b0).